lib/rubocop/cop/rails/delegate.rb

```diff
@@ -49,6 +49,7 @@ class Delegate < Cop
         def on_def(node)
           method_name, args, body = *node
           return unless trivial_delegate?(method_name, args, body)
+          return if safe_navigation?(body)
```

Comment:
As it's not clear why this is a problem this should probably be explain here and in the cop class documentation.